src/tests/Parser/Brewmaster/Stagger.test.js

```diff
@@ -0,0 +1,39 @@
+import Stagger from 'Parser/BrewmasterMonk/Modules/Core/Stagger';
+import { events, processEvents, FIGHT_END, TOTAL_STAGGERED } from './Fixtures/SimpleFight';
+
+const stagger = new Stagger();
+
+describe('Brewmaster.Stagger', () => {
```

Comment:
I think writing the test cases as if they're requirements would make for better tests, I don't think you should try to only have 1 assertion per test case, there's no gain in doing so. I'd propose the follow test cases:

- it *tracks the total amount of damage staggered*
- it *tracks the total amount of stagger damage taken*
- it *tracks the amount of stagger missing from the fight*